lib/elasticsearch/drain/cli.rb

```diff
@@ -48,42 +97,42 @@ def do_exit(code = 0, &block)
           exit code
         end
 
-        def instances
-          instances = active_nodes.map(&:ipaddress)
+        def instances(nodes)
+          instances = nodes.map(&:ipaddress)
           instances.join(' ')
         end
 
-        def adjusted_min_size
+        def adjusted_min_size(nodes)
           min_size = drainer.asg.min_size
           desired_capacity = drainer.asg.desired_capacity
-          if (desired_capacity - active_nodes.length) >= min_size # Removing the active_nodes won't violate the min_size
-            # Reduce the asg min_size proportionally
-            desired_min_size = (min_size - active_nodes.length) <= 0 ? 0 : (min_size - active_nodes.length)
-          else
-            # Removing the active_nodes will result in the min_size being violated
-            desired_min_size = desired_capacity - active_nodes.length
-          end
+          desired_min_size = if (desired_capacity - nodes.length) >= min_size # Removing the nodes won't violate the min_size
+                               # Reduce the asg min_size proportionally
+                               (min_size - nodes.length) <= 0 ? 0 : (min_size - nodes.length)
+                             else
+                               # Removing the nodes will result in the min_size being violated
+                               desired_capacity - nodes.length
+                             end
```

Comment:
Rubocop tells me this is right because I'm supposed to line up the `if`/`else`/`end` keywords.